config/desktop.json

```diff
@@ -97,7 +97,8 @@
 		"rubberband-scroll-disable": false,
 		"settings/security/monitor": true,
 		"settings/theme-setup": false,
-		"signup/social": false,
+        "signup/onboarding-flow": false,
+        "signup/social": false,
```

Comment:
The indentation here are 8 spaces which is inconsistent with the other lines. Let's make them consistent.